riposte-spi/src/test/java/com/nike/riposte/server/http/impl/RequestInfoImplTest.java

```diff
@@ -433,6 +433,7 @@ public void getContent_returns_null_if_request_is_not_complete_with_all_chunks()
             "/some/path/foo/bar/ | /some/path/{param1}/{param2}  | foo | bar",
             "/some/path/foo/bar  | /some/path/{param1}/{param2}/ | foo | bar",
             "/some/path/foo/bar/ | /some/path/{param1}/{param2}/ | foo | bar",
+            "/some/path/%E5%A5%BD/%E6%B5%8B%E8%AF%95/ | /some/path/{param1}/{param2}/ | %E5%A5%BD | %E6%B5%8B%E8%AF%95",
```

Comment:
Thanks for the details. I've updated the RequestInfo as you showed and changed the examples to be more simple and understandable.